openwisp_users/apps.py

```diff
@@ -71,3 +82,28 @@ def update_organizations_dict(cls, instance, **kwargs):
         cache.delete(cache_key)
         # forces caching
         user.organizations_dict
+
+    def create_organization_owner(cls, instance, **kwargs):
+        if not kwargs.get('created') or not instance.is_admin:
+            return
+        owner_model = load_model('openwisp_users', 'OrganizationOwner')
+        model = kwargs.get('sender')
+        org_users = model.objects.filter(organization=instance.organization)
+        is_first = org_users.first() == org_users.last() == instance
```

Comment:
is this how you're checking that the `OrganizationUser` just created is the first user in the org?

What about simply checking if there's any other `OrganizationUser` in this org apart from the one just created?

```python
OrganizationUser.objects.filter(organization=instance.organization).exclude(pk=instance.pk).exists()
```

This query should answer that question.